[IndexFilters] Expose prop to control PopoverOverlay (#9862)
Shopify/polaris: .changeset/gold-frogs-sniff.md

---
'@shopify/polaris': minor
---

> Exposes closeOnChildOverlayClick prop on IndexFilters to control the preventCloseOnChildOverlayClick prop on PopoverOverlay

Reviewer: ```suggestion
Added support for the `Filters``closeOnChildOverlayClick` prop to `IndexFilters`
```
